code/src/components/AlbumData.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import { Artists } from './Artists.js'
+import './albumData.css'
+
+export const AlbumData = (props) => {
+    return (
```

Comment:
I'm being super picky here, but don't forget to check your indentation settings in your editor - this file has ended up with 4 spaces (should be 2). See [this question on stack overflow](https://stackoverflow.com/c/technigo/questions/253) for a bunch of options to check.